src/StreamStats.jl

```diff
@@ -1,15 +1,15 @@
 module StreamStats
     import StatsBase, Distributions
+    import Base.mean, Base.std, Base.var, Base.cov, Base.cor
```

Comment:
This is just stylistic, but I really like annotating each specific method addition for `mean` as `Base.mean`. It's more verbose at the definition site, but that verbosity really improve local readability of code.